docs/source/cpp/compute.rst

```diff
@@ -1356,11 +1360,19 @@ For timestamps inputs with non-empty timezone, localized timestamp components wi
 * \(1) Outputs the number of the day of the week. By default week begins on Monday
   represented by 0 and ends on Sunday represented by 6. :member:`DayOfWeekOptions::week_start` can be used to set
   the starting day of the week using ISO convention (Monday=1, Sunday=7). Day numbering can start with 0 or 1
-  using :member:`DayOfWeekOptions::one_based_numbering` parameter.
+  using :member:`DayOfWeekOptions::count_from_zero` parameter.
 * \(2) First ISO week has the majority (4 or more) of it's days in January. ISO year
-  starts with the first ISO week.
+  starts with the first ISO week. ISO week starts on Monday.
   See `ISO 8601 week date definition`_ for more details.
-* \(3) Output is a ``{"iso_year": output type, "iso_week": output type, "iso_day_of_week":  output type}`` Struct.
+* \(3) First US week has the majority (4 or more) of it's days in January. US year
+  starts with the first US week. US week starts on Sunday.
+* \(4) Returns week number allowing for setting several parameters.
+  :member:`WeekOptions::week_starts_monday` sets what day does the week start with (Monday=true, Sunday=false).
+  :member:`WeekOptions::count_from_zero` sets that dates from current year that fall into last ISO week of
+  the previous year return 0 if true and 52 or 53 if false.
+  :member:`WeekOptions::first_week_is_fully_in_year` sets if the first week is fully in January (true), or is a
+  week that begins on December 29, 30, or 31 considered to be the first week of the new year (false)?
+* \(5) Output is a ``{"iso_year": output type, "iso_week": output type, "iso_day_of_week":  output type}`` Struct.
```

Comment:
```suggestion
* \(3) First US week has the majority (4 or more) of its days in January. US year
  starts with the first US week. US week starts on Sunday.
* \(4) Returns week number allowing for setting several parameters.
  If :member:`WeekOptions::week_starts_monday` is true, the week starts with Monday, else Sunday if false.
  If :member:`WeekOptions::count_from_zero` is true, dates from the current year that fall into the last ISO week
  of the previous year are numbered as week 0, else week 52 or 53 if false.
  If :member:`WeekOptions::first_week_is_fully_in_year` is true, the first week (week 1) must fully be in January;
  else if false, a week that begins on December 29, 30, or 31 is considered the first week of the new year.
* \(5) Output is a ``{"iso_year": output type, "iso_week": output type, "iso_day_of_week":  output type}`` Struct.
```

Reply:
Some wording tweaks just to make this flow a little better to me (feel free to accept/reject/reword as needed).